service.cronxbmc/gui.py

```diff
@@ -0,0 +1,217 @@
+import xbmc
+import xbmcgui
+import xbmcplugin
+import sys
+from urllib.parse import parse_qsl
+from resources.lib.cron import CronManager, CronJob
+from resources.lib.cron_utils import utils
+
+
+class CronGUI:
+    params = {}
+    context_url = "%s?%s"
+    plugin_url = 'Xbmc.RunPlugin(%s?%s)'

```

Comment:
Won't work, XBMC prefix have been dropped. Rename to RunPlugin only please

Reply:
Is that like a planned deprecation? It does still work in Kodi Matrix.